src/main/java/com/manardenza/dao/ReservationDaoImpl.java

```diff
@@ -0,0 +1,31 @@
+package com.manardenza.dao;
+
+import com.manardenza.entity.Hotel;
+import com.manardenza.entity.Reservation;
+
+import java.util.List;
+
+public class ReservationDaoImpl extends AbstractDao<Reservation> {
+
+    private static ReservationDaoImpl instance;
+
+    private ReservationDaoImpl() {
+    }
+
+    public static ReservationDaoImpl getInstance() {
+        if (instance == null) {
+            instance = new ReservationDaoImpl();
+        }
+        return instance;
+    }
+
+    @Override
+    public Reservation save(Reservation obj) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this method to simply inherit it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1185)